pages/linux/arecord.md

```diff
@@ -0,0 +1,16 @@
+# arecord
+
+> Sound recorder for ALSA soundcard driver.
+> More information: <https://linux.die.net/man/1/arecord>.
+
+- Record a snippet in "CD" quality (finish with Ctrl-C when done):
+
+`arecord -vv -f cd {{path/to/file.wav}}`
+
+- Record a snippet in "CD" quality, with a fixed duration of 10 seconds:
+
+`arecord -vv -f -d {{duration_in_seconds}} cd {{path/to/file.wav}}`
```

Comment:
This doesn't quite match the description. Perhaps a specific example would aid clarity here?
```suggestion
`arecord -vv -f -d {{10}} cd {{path/to/file.wav}}`
```

Also, are there any long forms of these arguments (e.g. `-f`, `-d`, etc)? We generally refer long-form arguments in tldr pages to aid clarity.

Reply:
Thanks for the review! Happy to change to long form, maybe everything except `-v`?

Reply:
b2a05e043ebccf79ff4a20bd10c3cd9ca44b236a